src/plasma/plasma_extension.c

```diff
@@ -71,6 +71,23 @@ PyObject *PyPlasma_create(PyObject *self, PyObject *args) {
   return PyBuffer_FromReadWriteMemory((void *) data, (Py_ssize_t) size);
 }
 
+PyObject *PyPlasma_hash(PyObject *self, PyObject *args) {
+  plasma_connection *conn;
+  object_id object_id;
+  if (!PyArg_ParseTuple(args, "O&O&", PyObjectToPlasmaConnection, &conn,
+                        PyObjectToUniqueID, &object_id)) {
+    return NULL;
+  }
+  unsigned char digest[DIGEST_SIZE + 1];
+  plasma_compute_object_hash(conn, object_id, digest);
+  digest[DIGEST_SIZE] = '\0';
+  if (strcmp(digest, "") == 0) {
+    Py_RETURN_NONE;
+  }
+  PyObject *digest_string = PyString_FromString(digest);
```

Comment:
I think it's actually better to have the digests be of type `unsigned char *` or `uint8_t *`. What's your reasoning for `int8_t *`?